### ISSUE TITLE
[issue]: Ventoy Can't find bootfile for UEFI only Image

### ISSUE THREAD

**Issue by CosmicFusion** [ventoy/Ventoy#2930]
### Official FAQ

- [X] I have checked the official FAQ.

### Ventoy Version

1.0.99

### What about latest release

Yes. I have tried the latest release, but the bug still exist.

### Try alternative boot mode

Yes. I have tried them, but the bug still exist.

### BIOS Mode

UEFI Mode

### Partition Style

GPT

### Disk Capacity

8GB

### Disk Manufacturer

Toshiba

### Image file checksum (if applicable)

None

### What happened?

  Hello I'm a developer of PikaOS our ISOs
  
  only support UEFI, and uses refind as bootloader, it works fine when flashing directly
  
  but with ventoy we get "No bootfile found for UEFI
  Maybe the image doesn't support x64 UEFI"
  
also the ISO: 
https://iso.pika-os.com/PikaOS-GNOME-4.0-amd64-v3-24.08.01-4.iso
https://iso.pika-os.com/PikaOS-GNOME-4.0-amd64-v3-24.08.01-4.md5
  
his is our ISO generator https://git.pika-os.com/images/live-iso-gnome-amd64-v3
  
and this is specifically how our efiboot.img that contains refind files is integrated into the ISO
  
`xorriso \
    -as mkisofs \
    -iso-level 3 \
    -V $LIVE_BOOT_VOLUME_NAME \
    -e --interval:appended_partition_2:all:: \
    -append_partition 2 0xef "$LIVE_BOOT_PATH/efiboot.img" \
    -J -joliet-long \
    -no-emul-boot \
    -isohybrid-gpt-basdat \
    -o "./output/$ISO_IMAGE".iso \
    "$LIVE_BOOT_DATA_PATH"`

If Ventoy could be adapted to support this system
or if anyone can help us become Ventoy compatible
that would be great!
Cosmo of PikaOS!

**Comment by monito-estratega** [comment]
Official FAQ
-
Checked.

Ventoy Version
-
1.0.99 (also tested with 1.0.97).

What about latest release?
-
Yep, 1.0.99 is the one atm.

Try alternative boot mode
-
Did it and got the same output error.

BIOS mode
-
UEFI.

Partition type
-
a.

Disk capacity
-
64 GB.

Disk manufacturer
-
Kingston.

Image file checksum
-
23628f5d78bb98c6c72b2b39c4f4056f

Image file download link
-
[https://iso.pika-os.com/PikaOS-GNOME-4.0-amd64-v3-24.08.01-4.iso](https://iso.pika-os.com/PikaOS-GNOME-4.0-amd64-v3-24.08.01-4.iso)

What happened?
-
I was checking if the new ISO of PikaOS 4 would boot for installation: it didn't for ol' trusty Ventoy, it did for full disk flash with Fedora Media Writer. It is to my understanding that the ISO can boot with no problems and normally flashed through Balena Etcher too.
Downloaded and checked the ISO. Sum is all right and it also shows correctly in the Ventoy verifier. Both in normal and grub2 mode the result is the same:
![iso2](https://github.com/user-attachments/assets/d528b70d-4a4d-42c3-b3d3-61319e33a610)

I am no expert, but hopes this can shine some light to the problem and help in any way or form.

Thanks for all, Ventoy team! Been using Ventoy since I switched to Lignux around 2022 <3.
